### PR TITLE
DEVEX-1387 documentation python2 -> python3

### PR DESCRIPTION
**Pull request by emiloslavsky**
DEVEX-1387 documentation python2 -> python3

### CODE REVIEW

File: src/python/scripts/dx-generate-dxapp [hunk 1]
```diff
@@ -330,6 +331,9 @@ def main():
     parser.add_argument(
         '--release', default="14.04",
         help="Distribution release to use for the machine image.")
+    parser.add_argument(
+        '--version', default="0",
+        help="Version of the application execution environment inside the runSpec block.")
```
Comment (commandlinegirl) [inline src/python/scripts/dx-generate-dxapp:336]: This name of the variable may conflict with "version" of the app, if we ever want to introduce it here. I'd change it into e.g. runspec-version.

File: src/python/scripts/dx-generate-dxapp [hunk 2]
```diff
@@ -128,6 +128,7 @@ class ArgumentParser(argparse.ArgumentParser):
                     }
                 },
                 "interpreter": self._args.interpreter,
+                "version": self._args.version,
```
Comment (commandlinegirl) [inline src/python/scripts/dx-generate-dxapp:131]:
```suggestion
                "version": self._args.runspec_version,
```

see comment below

Comment (emiloslavsky) [reply src/python/scripts/dx-generate-dxapp]: thanks, done!

File: src/python/scripts/dx-generate-dxapp (hunk 1 above)
Comment (emiloslavsky) [reply src/python/scripts/dx-generate-dxapp:336]: done

**Comment by emiloslavsky** [comment]
@commandlinegirl please take a look at the  code review fixes

**Comment by emiloslavsky** [comment]
Re: doc/examples/ cleanup - there's not task to do this.  Priority of doing this is low.

**Pull request merged**